manim/mobject/geometry.py

```diff
@@ -9,6 +9,8 @@
     "CurvedDoubleArrow",
     "Circle",
     "Dot",
+    "AnnotationDot",
+    "LabeledDot",
```

Comment:
```suggestion
    "LabelledDot",
```
Maybe I am missing something, do you mean a labelled dot?

Reply:
https://www.grammarly.com/blog/labeled-labelled/#:~:text=Labeled%20and%20labelled%20are%20both,preferred%20spelling%20in%20British%20English.

Both is correct.
Did we yet decide, if we want to use American, or British English?

Reply:
Oh that's a good point. I'd say American since most of the library is already in American.

Reply:
ok! Then I keep it like this.